src/main/resources/application.properties

```diff
@@ -21,3 +30,12 @@ management.endpoints.web.exposure.include= health, info
 management.health.key.enabled=diskspace, db
 management.endpoint.health.show-details= always
 
+
+
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=group4siit@gmail.com
+spring.mail.password=pnjlkyyroexhxsgm
```

Comment:
don't commit your email password in public git repos, substitute with ${EMAIL_PASS} enviornment variable

Reply:
and define it on your local machine